client/components/searchImage.jsx

```diff
@@ -3,31 +3,51 @@ import {Container, Row, Col} from 'reactstrap';
 import Gallery from "react-photo-gallery";
 import Carousel, { Modal, ModalGateway } from 'react-images';
 
-
 class searchImage extends Component {
     constructor(props) {
         super(props);
         this.state = {
-            //Initial Image
-            images: [
-                {src: 'client/src/InitImg.jpg', width: 4, height: 5},
-                {src: 'client/src/InitImg.jpg', width: 4, height: 5},
-                {src: 'client/src/InitImg.jpg', width: 4, height: 5},
-                {src: 'client/src/InitImg.jpg', width: 4, height: 5},
-                {src: 'client/src/InitImg.jpg', width: 4, height: 5},
-                {src: 'client/src/InitImg.jpg', width: 4, height: 5},
-                {src: 'client/src/InitImg.jpg', width: 4, height: 5}],
+            images: [{src: 'client/src/InitImg.jpg', width: 1, height: 1}],
             modalIsOpen: false,
-            selectedIndex: 0
+            selectedIndex: 0,
+            urls: []
         };
         this.viewSelectedImage = this.viewSelectedImage.bind(this);
         this.closeSelectedImage = this.closeSelectedImage.bind(this);
     }
 
+    componentDidMount() {
+        var data = {target: 'IZONE'};
+        fetch('http://localhost:5000/getImageUrl', {
+            credentials: 'same-origin',
+            method: 'POST', 
+            body: JSON.stringify(data), 
+            headers: new Headers({
+                'Content-Type' : 'application/json',
+                'Accept': 'application/json'
+            })
+        }).then(response => response.json()
+        ).then((result) => {
+            console.log(result);
+            var tmp = [];
+            this.setState(
+                {urls: result}, () => {
+                    this.state.urls.map(path => {
+                        var format = {src: `${path}`, width: 1, height: 1};
```

Comment:
please be more thoughtful when picking a variable name. 
Always remember that someone will be looking at your code

use image